### ISSUE TITLE
Validation Error in the test_arxiv_workflow.py

### ISSUE THREAD

**Issue by rikirenz** [inspirehep/inspire-next#1763]
The test `test_harvesting_arxiv_workflow_accepted` in `test_arxiv_workflow.py` has the `@pytest.mark.xfail` decorator. I removed it and I executed the test. 
The line that is triggering the error is the: https://github.com/inspirehep/inspire-next/blob/master/tests/integration/workflows/test_arxiv_workflow.py#L391

The error is the following:
`ValidationError: u'2014' is not of type u'integer'`

I have tried to add this line before the saving record operation:
``` python
[...]
obj.data['publication_info'][0]['year'] = 2014 #  -> this line
obj.save()
db.session.commit()
[...]
```
and the test works.

I think is a problem with the new JSON Schema.

**Comment by jacquerie** [comment]
> I think is a problem with the new JSON Schema.

It's fine that the year is an integer rather than a string, so that [we can predicate reasonable bounds on it](https://github.com/inspirehep/inspire-schemas/blob/8d840a6cd83d603b0ce5a12c36144266eaa8b660/inspire_schemas/records/hep.json#L669-L673).

So there must be someone who's putting a string there. Is it DoJSON? Is it some task in the workflow? To know that, you will have to print `record_json['publication_info'][0]['year']` just after [this line](https://github.com/inspirehep/inspire-next/blob/d124c827a35c5c5ad7670880eda723bf5258f420/tests/integration/workflows/test_arxiv_workflow.py#L353).

**Comment by jacquerie** [comment]
Fixed in #1765.
